ledger-tool/src/main.rs

```diff
@@ -1056,8 +1055,9 @@ fn main() {
         .default_value(SnapshotVersion::default().into())
         .help("Output snapshot version");
 
-    let default_max_full_snapshot_archives_to_retain =
-        &DEFAULT_MAX_FULL_SNAPSHOT_ARCHIVES_TO_RETAIN.to_string();
+    // Use std::usize::MAX for maximum_*_snapshots_to_retain such that
+    // ledger-tool commands will not remove any snapshots by default
```

Comment:
wdyt about defining `DEFAULT_MAX_FULL_SNAPSHOT_ARCHIVES_TO_RETAIN` locally (to ledger-tool) as `usize::MAX`, put the comment there, then leave the stringifications untouched?  gaol being to retain self-documentation

Reply:
Yeah, we could do that. This change has already landed in master and v1.14, so assuming we'd want to adjust in those as well.

Reply:
Addressing in https://github.com/solana-labs/solana/pull/30745 - closing this PR out as I will BP that one

Reply:
ah sry, didn't realize this was a bp when i was initially reviewing

Reply:
Ehh, no biggie as long as I can burden you with the extra reviews 😅